ahemap/settings_shared.py

```diff
@@ -10,8 +10,8 @@
 
 locals().update(common(project=project, base=base))
 
-if distro.linux_distribution()[0] == 'Ubuntu':
-    if distro.linux_distribution()[1] == '16.04':
+if distro.id() == 'ubuntu':
+    if distro.version() == '16.04':
```

Comment:
These Ubuntu versions are so old, we can probably just remove this entire `if` block.

```